ground/src/test/java/com/google/android/ground/ui/datacollection/DataCollectionTestData.kt

```diff
@@ -28,11 +28,13 @@ import java.util.*
 import java8.util.Optional
 
 object DataCollectionTestData {
```

Comment:
Is this class shared between tests? 

We already have a FakeData class that contains generic objects. Tests use that as baseline and modify then according to their needs. Would it possible to remove this in favor of `FakeData`?

Everything which is specific to the test should live inside that. Only common test data should be moved to FakeData.

Reply:
This class is now only used by this test.

I think we could remove this and use FakeData, but FakeData doesn't have a Submission, so I think we would either have to add that to the DataCollectionFragmentTest or to FakeData.

Let me know which option you prefer!

Reply:
Adding a dummy submission object to FakeData sounds better.

Reply:
Sounds good, moved to FakeData